codalab/bundles/run_bundle.py

```diff
@@ -36,6 +36,11 @@ class RunBundle(DerivedBundle):
     METADATA_SPECS.append(MetadataSpec('request_priority', int, 'Job priority (higher is more important). Negative '
                                                                 'priority bundles are queued behind bundles with no specified priority.', default=None))
     METADATA_SPECS.append(MetadataSpec('request_network', bool, 'Whether to allow network access.', default=False))
+    METADATA_SPECS.append(
+        MetadataSpec('cpu_usage', float, 'Proportion of current CPU usage for a running bundle. This field is only relevant for running bundles. (e.g., 0.24)', default=0.0, generated=True))
+    METADATA_SPECS.append(
+        MetadataSpec('memory_limit', int, 'How much memory this container can use at any given time.', default=0, generated=True))
```

Comment:
sorry, still not clear what this means. Does memory_limit change during the duration of a run? It sounds like it might be a constant value if it's "how much memory this container **can use**"

Reply:
https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory

according to docker docs "The maximum amount of memory the container can use.". I believe that it's a constant.

Reply:
Huh, well if it's just a constant, then maybe we should just remove it? See #3003 for the original issue of adding docker stats; not sure if a constant max memory value is really that useful.

Reply:
you are right.

I am not sure why yibo wanted to have `memory_limit` as a statistic. A more appropriate statistic would be the memory usage, so that would be `memory_usage/memory_stats`. I'll make this change.